### DISCUSSION TITLE
Some of the examples show up very small

### DISCUSSION THREAD

**Discussion by rnikander** [ocornut/imgui#7365] [New Users Build/Link/Run issues ONLY!]
I'm new to ImGui. I ran the example `example_glfw_vulkan` and it shows up very small on my laptop screen. I'm guessing width and height are 1/2 what they are meant to be. Not sure if that's a bug.

**Comment by ocornut** [comment]
It happens if the application is DPI aware (full resolution) but doesn't scale it contents manually.
An application that is not made DPI aware will be scaled by Windows, looks bigger but blurry.

A real application would likely want to load a font at double size to benefit from the higher resolution.
https://github.com/ocornut/imgui/blob/master/docs/FAQ.md#q-how-should-i-handle-dpi-in-my-application

I'm hoping this year that we'll sort of all examples and backends to make things nicer.
